database/cassandra/cassandra.go

```diff
@@ -296,16 +296,71 @@ func (c *Cassandra) ensureVersionTable() (err error) {
 		}
 	}()
 
-	err = c.session.Query(fmt.Sprintf("CREATE TABLE IF NOT EXISTS %s (version bigint, dirty boolean, PRIMARY KEY(version))", c.config.MigrationsTable)).Exec()
+	err = c.createVersionTable()
 	if err != nil {
 		return err
 	}
 	if _, _, err = c.Version(); err != nil {
+		var count int8
+		err = c.session.Query(`SELECT COUNT(*) FROM system_schema.columns WHERE keyspace_name = '` + c.config.KeyspaceName + `' AND table_name = '` + c.config.MigrationsTable + `' AND column_name = 'dummy'`).Scan(&count)
```

Comment:
I've put it here so in normal circumstances, this query is only performed if there is an error (which might indicate the table needs an update). Moving this in `createVersionTable` would mean there'd always be 2 calls done instead of 1